viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -933,6 +1000,13 @@ extern int viam_carto_get_position_response_destroy(
 
 extern int viam_carto_get_point_cloud_map(
     viam_carto *vc, viam_carto_get_point_cloud_map_response *r) {
+    if (vc == nullptr) {
```

Comment:
This adds error handling for preventing crashes if null pointers are passed to these C functions which I forgot to add in previous PRs.